dev/generate_bom.py

```diff
@@ -277,7 +277,7 @@ def publish_bom(self, bom_path):
     result = run_quick('hal admin publish bom --color false --bom-path {0}'
                        .format(bom_path))
     if result.returncode != 0:
-      print "'hal admin publish bom' command failed with: \n{0}\n exiting...".format(result.stdout)
+      print "'hal admin publish bom' command failed with: \n{0}\nexiting...".format(result.stderr)
```

Comment:
@ewiseblatt correct me if I'm wrong, isn't `stderr` piped through `stdout` in `run_quick`?